tools/osbuilder/rootfs-builder/ubuntu/Dockerfile.in

```diff
@@ -3,32 +3,50 @@
 # SPDX-License-Identifier: Apache-2.0
 
 ARG IMAGE_REGISTRY=docker.io
+
+# Install skopeo which is not included in 20.04 release
+# This can be removed when we upgrade the base to 22.04 release
+FROM ${IMAGE_REGISTRY}/golang:1.18 AS skopeo
+@SET_PROXY@
+WORKDIR /skopeo
+ARG SKOPEO_VERSION
```

Comment:
Sorry if I've missed it in this PR, but in order to set this value I added https://github.com/kata-containers/kata-containers/pull/6104/commits/e777860a0c999c269be1c6fbf589c1e957804067 in `CCv0` which built on some of the earlier work that Jakob did to add skopeo to versions.yaml in https://github.com/stevenhorsman/kata-containers/commit/13f6418c46c65764eba2fda4e8a33d62c5ac852b

Without it I'm not sure we are going to know what version of skopeo to use here?

Reply:
Fixed.